examples/plugins/BUILD.bazel

```diff
@@ -18,7 +18,6 @@ swc_plugin(
     name = "npm_plugin",
     srcs = [
         # reference the location where the "@swc/plugin-transform-imports" npm package was linked in our root Bazel package.
-        ":node_modules/@swc/plugin-transform-imports",
```

Comment:
The latest `@swc/plugin-transform-imports` adds a dependency on `@swc/counter`, I think just for tracking metrics, but we need a single file and currently rules_swc [assumes it's the first file](https://github.com/aspect-build/rules_swc/blob/v2.0.1/swc/private/swc.bzl#L239).

I think that assumption is a different issue but this works around it for this example.